src/modules/orders/index.ts

```diff
@@ -5,21 +5,97 @@ import OrderRepository from './order.repository'
 import { OrderType } from './order.entity'
 import { OrderDto } from './order.dto'
 import DealRepository from './deal.repository'
+import {
+  parseOrderCell,
+  parseOrderData,
+  bigIntToUint128Le,
+  SUDT_TX_HASH,
+  signAndSendTransaction,
+  DEFAULT_NODE_URL,
+  formatOrderData,
+  SECP256K1_CODE_HASH,
+  SECP256K1_TX_HASH,
+  ORDERBOOK_TX_HASH,
+} from '../../utils'
 import { Deal, DealStatus } from './deal.entity'
-import { logger, parseOrderCell } from '../../utils'
+import fs from 'fs'
+import CKB from '@nervosnetwork/ckb-sdk-core'
+import { getTransactionSize } from '@nervosnetwork/ckb-sdk-utils/lib/sizes'
+import { Indexer, CellCollector } from '@ckb-lumos/indexer'
```

Comment:
Imports from `node_modules` should be placed above those from modules of the project.